community/iputils/build

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh -e
+
+export DESTDIR="$1"
+
+patch -p1 < fix-setuid-redeclared.patch
+
+meson \
+    --prefix=/usr \
+    -DUSE_CAP=true \
+    -DNO_SETCAP_OR_SUID=true \
+    -DUSE_IDN=true \
```

Comment:
`-DUSE_IDN=false` and `libidn` can be removed.

Reply:
But then we can't use setcap in post-install?

Reply:
Yes we can. We're removing libidn and not libcap.